.github/pull_request_template.md

```diff
@@ -1,14 +1,14 @@
-# Description
+<!---
+Thanks for contributing to the Amplitude JavaScript SDK! 🎉
```

Comment:
The same question about the pr template. 
Not limited to javascript sdk.

Reply:
Yeah since there is only one, i think we should name this with something more generic

Reply:
Agree.

Reply:
Alright i just said "Amplitude TypeScript repository"